add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/java/dev/flutter/multipleflutters/DoubleFlutterActivity.kt

```diff
@@ -20,6 +20,10 @@ class DoubleFlutterActivity : FragmentActivity(), EngineBindingsDelegate {
         EngineBindings(activity = this, delegate = this, entrypoint = "bottomMain")
     }
     private val numberOfFlutters = 2
+    private var engineCountStart = 0
```

Comment:
Could this be `private val engineCountStart = engineCount`? We need to make sure the value never changes through future refactors so the onDestroy start number is right and not leak.